draft-selander-lake-authz.md

```diff
@@ -480,17 +494,28 @@ V performs the normal EDHOC verifications of message_3. V may retrieve CRED_U fr
 ## Authenticator <-> Authorization Service (V <-> W) {#V-W}
 
 It is assumed that V and W have set up a secure connection, W has accessed the authentication credential CRED_V to be used in the EDHOC session between V and with U, and that W has verified that V is in possession of the private key corresponding to CRED_V, see {{domain-auth}} and {{authz-server}}.
+V and W run the Voucher Request/Response protocol over the secure connection.
+
 
-V and W run the Voucher Request/Response protocol over the secure connection. The hash of EDHOC message_1, H(message_1), acts as session identifier and binds together instances of the two protocols (U<->V and V<->W).
 
 ### Voucher Request {#voucher_request}
 
 #### Processing in V
 
-V uses H(message_1) as a session identifier associated to this connection with W. If the same value of H(message_1) is already used for a connection with this or other W, the protocol SHALL be discontinued.
+V sends the voucher request to W.
+The Voucher Request SHALL be a CBOR array as defined below:
 
-V sends the voucher request to W. The Voucher Request SHALL be the CBOR array \[false, message_1\] where "false" is the CBOR simple value with encoding 0xf4.
+~~~~~~~~~~~
+Voucher_Request = [
+    message_1:      bstr,
+    ? opaque_state: bstr
+]
+~~~~~~~~~~~
+
```

Comment:
The reason for "false" was to avoid potential confusion with a message_1 intended from V to W (like we have for PoP). If there is not opaque_state, do you see any risk for this confusion?

Reply:
Content-Format of the forwarded message, according to PR #17, will be set to "application/fwd-edhoc+cbor-seq", which should distinguish it from a regular EDHOC message. I think that should be enough to distinguish it. Plus the endpoint is different, isn't it? Here V POSTs to /voucherrequest. In the PoP case, it will POST to /edhoc.

Reply:
Thinking more about this. Yes, this Content-Format renaming make sense.

Reply:
As does removing "false".